sale_last_price_info/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+{
+    "name": "Product Last Price Info - Sale",
+    "version": "13.0.2.0.0",
+    "author": "OdooMRP team, "
```

Comment:
Please change this to `AvanzOSC, Tecnativa`, which are the companies that are now